playground/next15/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "next15",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "15.0.3",
+    "react": "19.0.0-rc-66855b96-20241106",
+    "react-dom": "19.0.0-rc-66855b96-20241106"
+  },
+  "devDependencies": {
+    "@storyblok/react": "latest",
```

Comment:
```suggestion
    "@storyblok/react": "workspace:^",
```

Reply:
Hi @edodusi I also got a hydration error by running the `playground-next` script. Can we live with it?

![Screenshot 2024-12-09 at 10 23 50](https://github.com/user-attachments/assets/8d71d465-37d2-4ee7-994f-dcc3278dc748)